tests/common/jormungandr/logger.rs

```diff
@@ -0,0 +1,34 @@
+use std::fs::File;
+use std::io::{BufRead, BufReader};
+use std::path::PathBuf;
+
+#[derive(Debug)]
+pub struct JormungandrLogger {
+    pub log_file_path: PathBuf,
+}
+
+impl JormungandrLogger {
+    pub fn new(log_file_path: PathBuf) -> Self {
+        JormungandrLogger { log_file_path }
+    }
+
+    pub fn get_lines_with_error(&self) -> Vec<String> {
+        let lines = self.get_lines_from_log();
+        lines
+            .iter()
+            .filter(|n| n.to_uppercase().contains("ERROR"))
```

Comment:
It would be better and faster to use a regex, which 1) can be case-insensitive without you needing to fiddle with the string; 2) can match a particular location in the line, like `^ERROR`.

Reply:
In fact, we have a machine-readable output format, `stderr_json`. Why not use that and parse the severity level as a JSON field?